<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionView</class>
 <widget class="QDialog" name="SessionView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1759</width>
    <height>999</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SessionView</string>
  </property>
  <widget class="QPushButton" name="finishButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tickButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tick</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previousButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>1731</width>
     <height>671</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="redrawButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Redraw</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="tickSpinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>1670</x>
     <y>960</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="graphPlotView" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>750</y>
     <width>1141</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>1670</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lastButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Last</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="renderCheckBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Render</string>
   </property>
  </widget>
  <widget class="QComboBox" name="changeStateComboBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="changeStateSpinbox">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="changeStateButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change State</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addVertexButton">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>10</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Vertex</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sourceSpinbox">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>10</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="targetSpinbox">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>40</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>10</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>40</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Target</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addEdgeButton">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Edge</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeEdgeButton">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Edge</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vertex</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>0</y>
     <width>20</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>0</y>
     <width>20</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="addVertextStateComboBox">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>10</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>0</y>
     <width>20</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="removeVertexSpinbox">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>40</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="removeVertexButton">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>40</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Vertex</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vertex</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
